packages/client/components/GroupingKanban.tsx

```diff
@@ -38,36 +38,44 @@ export type SwipeColumn = (offset: number) => void
 
 const GroupingKanban = (props: Props) => {
   const {meeting, phaseRef} = props
-  const {id: meetingId, reflectionGroups, phases, spotlightReflection} = meeting
+  const {reflectionGroups, phases, spotlightReflection} = meeting
   const reflectPhase = phases.find((phase) => phase.phaseType === 'reflect')!
   const reflectPrompts = reflectPhase.reflectPrompts!
   const reflectPromptsCount = reflectPrompts.length
-  const spotlightReflectionRef = useRef<HTMLDivElement | null>(null)
-  const {onOpenSpotlight, onCloseSpotlight} = useSpotlightSimulatedDrag(meeting)
-  const [flipRef, flipReverse] = useFlip({
-    firstRef: spotlightReflectionRef
-  })
   const [callbackRef, columnsRef] = useCallbackRef()
   useHideBodyScroll()
+  const dragIdRef = useRef<string>()
+  const {onOpenSpotlight, onCloseSpotlight} = useSpotlightSimulatedDrag(meeting, dragIdRef)
+
   const closeSpotlight = () => {
+    sourceCloneRef.current = null
     onCloseSpotlight()
   }
-  const {closePortal, openPortal, modalPortal} = useModal({
+
+  const {closePortal, openPortal, modalPortal, portalStatus} = useModal({
     onClose: closeSpotlight,
     id: 'spotlight'
   })
+  const {sourceRef, sourceCloneRef} = useAnimatedSpotlightSource(
+    portalStatus,
+    spotlightReflection?.id,
+    dragIdRef
```

Comment:
`StartDraggingReflectionMutation` needs to happen after the cloning as the reflection height will be altered. So, I've moved the mutation from `useSpotlightSimulateDrag` to `useAnimatedSpotlightSource`. The downside is we need to share `dragIdRef` across both hooks.

Reply:
the flow is:
- reflection is 88px tall in the kanban
- if you trigger start dragging, then the height will reduce if it's the only one in the group (see useCollapsePlaceholder)
-  so, we need to clone first before calling start drag

Reply:
i think this is good! it's the smallest nit of wondering if we could combine all the "opening" logic into 1 chunk, but we can think about refactoring later

Reply:
One possible solution which might mean we can clone the reflection after executing `StartDraggingReflectionMutation` is adjusting the `isClipped` logic - see [this comment](https://github.com/ParabolInc/parabol/pull/5610/files#r742380685).

But, as you mention in your comment, I'd prefer to merge this PR and refactor later.